### ISSUE TITLE
Says "The request timed out" when join newwork with my Network ID, retry doesn't work, have to quit.

### ISSUE THREAD

**Issue by mchotdograp** [zerotier/ZeroTierOne#1296]
Says "The request timed out" when join newwork with my Network ID, retry doesn't work, have to quit.

 - OS: Mac Mojave 10.14.6
 - ZeroTier Version 1.6
  
It was OK with v1.4.6, then I updated the client to v1.6, and now it's not OK.

**Comment by glimberg** [comment]
Give this a try:  http://download.zerotier.com/RELEASES/1.6.1/dist/ZeroTier%20One.pkg

**Comment by mchotdograp** [comment]
Thanks! I tried this 1.6.1 Mac zerotier and connected to the network!
But next thing is my RDP on Mac cannot connect to one of the Win10 node of my network, while RDP on another node(Win7) can connect to that Win10 node. 
So I tried ping.
On my Mac, I pinged the managed IP of the Win10 node, turned out timeout, but when I pinged the Physical IP of the Win10 node, it worked!
So I did the same thing on the win7 node, both IPs of the Win10 node worked!
Now I can say that there must be something wrong with the new Mac version Zerotier, given that the old version works fine.
